core/domain/feedback_jobs_one_off.py

```diff
@@ -66,3 +66,33 @@ def map(item):
     @staticmethod
     def reduce(key, instance_ids):
         yield (key, instance_ids)
+
+
+class PopulateMessageCountOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for populating the message count field."""
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        """Returns a list of datastore class references to map over."""
+        return [feedback_models.GeneralFeedbackThreadModel]
+
+    @staticmethod
+    def map(thread):
+        if thread.message_count is None:
+            # Assigning the value of message_count if it is None.
+            thread.message_count = feedback_models.GeneralFeedbackMessageModel.get_message_count(thread.id)#pylint: disable=line-too-long
```

Comment:
You can break after the "=" sign:

```
            thread.message_count = (
                feedback_models.GeneralFeedbackMessageModel.get_message_count(
                    thread.id))
```

Also @apb7 @lilithxxx it looks like there should be a space before and after the '#' in the pylint pragma. Should the lint checks catch this?